src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -316,7 +288,156 @@ To ensure your CSS files are accessible, add the `crossorigin="anonymous"` attri
 Here's an example of adding the `crossorigin="anonymous"` attribute:
 
 ```
- <link rel="stylesheet" href="assets.yoursite.com/styles.css" crossorigin="anonymous">
+<link rel="stylesheet" href="assets.yoursite.com/styles.css" crossorigin="anonymous">
 ```
 </Collapser>
 </CollapserGroup>
+
+## Manually record session replays [#manual-replays]
+
+If needed, you can programmatically invoke a session replay using these two API methods:
```

Comment:
(Outside of the scope of this PR) This feels like an edge use case and a good candidate for moving to its own doc, I think. Also, it would be great to have more context about why someone would want to do this (but maybe we don't know yet?)